.github/workflows/license-generation.yml

```diff
@@ -84,13 +88,13 @@ jobs:
   create-licenses:
 
     runs-on: ubuntu-latest
-  
+
     container:
-      image: zowe-docker-release.jfrog.io/ompzowe/zowecicd-license-base:latest
+      image: zowe-docker-snapshot.jfrog.io/ompzowe/zowecicd-license-base:test-timothy
```

Comment:
@MarkAckert I'm happy to update the `zowe-docker-release` latest tag to include the updated ORT and pnpm install step if you'd like 😋